src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml

```diff
@@ -114,7 +114,7 @@
         var usedUnresolvedAssembly = Model.GetUnresolvedAssemblies(assembly);
         if (usedUnresolvedAssembly.Count > 0)
         {
-            <table>
+            <table aria-labelledby="@assembly.SourceAssembly">
```

Comment:
Looks like this table is for missing assemblies. What if instead of adding aria-labelledby="@assembly.SourceAssembly", adding a caption with value of @LocalizedStrings.MissingAssemblies? Certainly, that could seems duplicated with the table header  of LocalizedStrings.MissingAssemblies .